test/fixtures/events/no-event-loop.js

```diff
@@ -2,5 +2,5 @@ global.process = { __proto__: process, pid: 123456 }
 Date.now = function () { return 1459875739796 }
 require('os').hostname = function () { return 'abcdefghijklmnopqr' }
 var pino = require(require.resolve('./../../../'))
-var log = pino({extreme: true})
+var log = pino(pino.extreme())
```

Comment:
why do we pass `pino.extreme` to `pino` here?.

Reply:
That's the new API. `pino.extreme` is a utility method to create a destination stream.